gzsitl/CMakeLists.txt

```diff
@@ -1,21 +1,17 @@
 cmake_minimum_required(VERSION 2.8 FATAL_ERROR)
 
-set(CUSTOM_COMPILE_FLAGS "-Wall")
-set(LDFLAGS "-z noexecstack -z relro -z now")
-set(CFLAGS "-fstack-protector-all -fPIE -fPIC -O2 -D_FORTIFY_SOURCE=2")
+set(CUSTOM_COMPILE_FLAGS "${CUSTOM_COMPILE_FLAGS} -Wall")
+set(LDFLAGS "${LDFLAGS} -z noexecstack -z relro -z now")
+set(CFLAGS "${CFLAGS} -fstack-protector-all -fPIE -fPIC -O2 -D_FORTIFY_SOURCE=2")
```

Comment:
Apparently yes, according to [cmake](https://cmake.org/cmake/help/v3.0/command/add_library.html). However, -fPIC is an intel requirement for both shared and static libraries, and since the cmake could be changed to make the library static in some cases, I think it's safer just to keep the flag explicit.